### DISCUSSION TITLE
Still reachable como memory leak

### DISCUSSION THREAD

**Discussion by franmolinam** [IIC2133-PUC/2024-2#186] [Tarea 3] [answered]
Hola! Estoy probando si hay leaks de memoria con Valgrind y en la parte de definitely lost no hay ningún byte, tampoco en la parte de indirectly lost ni en possibly lost. Mi pregunta es si la parte de Leak Summary que se llama still reachable cuenta como un leak de memoria si hay bytes ahí. Estuve leyendo y normalmente no se considera, porque el programa debería liberar eso solo, pero no se cual será el criterio de evaluación. Gracias!

**Comment by JavierLarre** [comment, marked as answer]
Si, es en general buena práctica liberar la memoria no usada y nosotros, al corregir, lo consideramos memory leak. Valgrind debe retornar `All heap blocks were freed -- no leaks are possible`. Suerte!
